### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by ErrorPro**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-module.exports = function (babel) {
+/* eslint-disable no-var */
+module.exports = function(babel) {
```
Comment (vslinko) [inline lib/index.js]: Please add space between comment and `module.exports`

File: examples/with-imported-react.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 
 export default class Component {
+  static displayName() {}
```
Comment (vslinko) [inline examples/with-imported-react.js]: Why you added this?

**Comment by vslinko** [comment]
@ErrorPro reviewed

File: examples/with-imported-react.js (hunk 2 above)
Comment (ErrorPro) [inline examples/with-imported-react.js]: Because i've had a error about missing display name property in definition class name Component.

Comment (vslinko) [inline examples/with-imported-react.js]: You should change `react/display-name` eslint rule from 2 to 0

**Pull request merged**